pkg/proc/native/support_sentinel_linux.go

```diff
@@ -1,6 +1,6 @@
 // This file is used to detect build on unsupported GOOS/GOARCH combinations.
 
-//go:build linux && !amd64 && !arm64 && !386
-// +build linux,!amd64,!arm64,!386
+//go:build linux && !amd64 && !arm64 && !386 && !ppc64le
```

Comment:
What's the story going to be with CI? Are we going to have a ppc64 builder? Otherwise we should do the same thing we did with windows/arm64 and put it behind a build tag I think.

Reply:
I requested a PPC64LE machine to put it into the CI system. I already have it.

Reply:
What's the status on getting TeamCity agent installed there?